src/FakeAuth/HttpClientExtensions.cs

```diff
@@ -2,15 +2,14 @@
 using System.IO;
 using System.Net.Http;
 using System.Security.Claims;
-using FakeAuth.Internal;
 
-namespace FakeAuth.Tests
+namespace FakeAuth
```

Comment:
This change I disagree with. The reason I put it into a `.Tests` namespace is because it's perfectly valid for production code to use `HttpClient`, say, to call some other service. I wanted this to be in a different namespace because it doesn't make much sense for anything but the tests to use `SetFakeAuthClaims(...)`. I thought seeing a `.Tests` namespace in prod code ought to raise eyebrows and call out what's likely a mistake.

Reply:
Makes sense.. I just don't think anything from FakeAuth namespace should ever be run in production code, for http client or otherwise. Actually named in Fake Auth, instead of ZeroAuth or something like that to make it clear this for Faking things.. 

Do you think someone might leave FakeAuth namespace in place for prod? If so, I'm happy to put the HttpClient extenctions back in a Test namespace. To me though, it feels like a Moq library utility needing to specify an additional namespace to say it's only for tests, but I get it.

Reply:
Despite your intentions, imma use this thing in prod because it makes testing so much easier 😁 Bear in mind we've got two testing scenarios - localhost manual testing using whatever the default claims are, and automated testing with the mocked claims in the header. The former means (likely) FakeAuth is directly referenced by the prod code; the latter, we've got opportunity to completely isolate FakeAuth from the prod code. So, if the intent is to do any kind of manual testing via the browser that relies on the configured default claims, FakeAuth will be a direct dependency of the prod code. That was my assumption/thinking, anyway.

Reply:
Write now.. the handler will throw an exception if you're not coming from localhost.  Worth discussing, but this has been around from the beginning... 

https://github.com/calebjenkins/FakeAuth/blob/Develop/src/FakeAuth/FakeAuthHandler.cs#L33

Reply:
Write now.. the handler will throw an exception if you're not coming from localhost.  Worth discussing, but this has been around from the beginning... 
https://github.com/calebjenkins/FakeAuth/blob/Develop/src/FakeAuth/FakeAuthHandler.cs#L33

Reply:
OR if ://localhost isn't in the URL.. I suppose someone could append that to the end of whatever they are passing. 🤔
¯\_(ツ)_/¯

Reply:
I've thought of adding a config/options flag called something like `ByPassLocaHostCheck_DontMessWithThis` or something.. but have avoided that so far. 😂

Reply:
Yeah, I think that ought to be configurable, with the current behavior as the default. I'll probably get a PR for that together in the near future.